<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_employee</class>
 <widget class="QMainWindow" name="Edit_employee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit an Employee</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/icon/edit.png</normaloff>img/icon/edit.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(195, 151, 245);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0,0,0,0,0,0,0">
    <item row="1" column="0">
     <widget class="QPushButton" name="image_effect_btn">
      <property name="font">
       <font>
        <family>MRT_Zoheir</family>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>image effectðŸŽˆ</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="nationalcode">
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(33, 100, 84);</string>
      </property>
      <property name="placeholderText">
       <string>National Code</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QComboBox" name="year">
      <property name="currentText">
       <string comment="1970"/>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Perpetua</family>
        <pointsize>26</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(6, 255, 198);
color: rgb(225, 6, 139);</string>
      </property>
      <property name="text">
       <string>Edit Employee</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QComboBox" name="month">
      <item>
       <property name="text">
        <string>Jan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Feb</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Apr</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>May</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jun</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jul</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Aug</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sep</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oct</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nov</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dec</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="9" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QPushButton" name="cancel_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 14, 14);</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="done_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(38, 250, 14);</string>
        </property>
        <property name="text">
         <string>Done</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="lastname">
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(33, 100, 84);</string>
      </property>
      <property name="placeholderText">
       <string>Last Name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="firstname">
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(33, 100, 84);</string>
      </property>
      <property name="placeholderText">
       <string>First Name</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QComboBox" name="day">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>24</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>26</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>27</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>28</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>29</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>31</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
